data/en/getbasetagdata.json

```diff
@@ -6,21 +6,33 @@
 	"related":["getBaseTagList"],
 	"description":" Used within a custom tag. Finds calling (ancestor) tag by\n name and accesses its data.",
 	"params": [
-		{"name":"tagname","description":"","required":true,"default":"","type":"string","values":[]},
-		{"name":"instancenumber","description":"","required":false,"default":"","type":"numeric","values":[]}
+		{"name":"tagname","description":"Specify the parent tag name, starting with CF_","required":true,"default":"","type":"string","values":[]},
+		{"name":"instancenumber","description":"Specify the nth-level ancestor to retrieve variables from","required":false,"default":"1","type":"numeric","values":[]}
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"4", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-e-g/getbasetagdata.html"},
+		"coldfusion": {"minimum_version":"4", "notes":"CF2018+ Changed parameter name instanceNumber to ancestorLevels.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-e-g/getbasetagdata.html"},
```

Comment:
I'm not sure this note is useful since parameter names were not used before CF2018.

Reply:
Oh, I didn't know that! I'll remove that note.

Reply:
Wait - it's useful because ACF and Lucee now have different attribute names, so you can't run `getBaseTagData( tagname = "myCustomTag", instanceNumber="5" )` on ACF 2018+.

Reply:
Lucee supports all of the same parameter names, sometimes as aliases.

Reply:
By the way, I created #898 to address this kind of cross-compatibility issue in CFDocs.